src/cinder/Text.cpp

```diff
@@ -675,21 +678,24 @@ vec2 TextBox::measure() const
 Surface	TextBox::render( vec2 offset )
 {
 	createLines();
-	
+
 	float sizeX = ( mSize.x <= 0 ) ? mCalculatedSize.x : mSize.x;
 	float sizeY = ( mSize.y <= 0 ) ? mCalculatedSize.y : mSize.y;
+
 	sizeX = math<float>::ceil( sizeX );
 	sizeY = math<float>::ceil( sizeY );
-	
+
 	Surface result( (int)sizeX, (int)sizeY, true );
 	ip::fill( &result, mBackgroundColor );
 	::CGContextRef cgContext = cocoa::createCgBitmapContext( result );
 	::CGContextSetTextMatrix( cgContext, CGAffineTransformIdentity );
-	
+
 	for( vector<pair<shared_ptr<const __CTLine>,vec2> >::const_iterator lineIt = mLines.begin(); lineIt != mLines.end(); ++lineIt ) {
-		::CGContextSetTextPosition( cgContext, lineIt->second.x + offset.x, sizeY - lineIt->second.y + offset.y );
+		int curIndex = lineIt - mLines.begin();
+		::CGContextSetTextPosition( cgContext, lineIt->second.x + offset.x, sizeY - lineIt->second.y - (mLeading * curIndex) + offset.y );
```

Comment:
Looks a bit funny to me that we subtract the additional leading. Does it definitely work?